pkgs/development/tools/yamlpath/default.nix

```
@@ -46,5 +46,6 @@ python3.pkgs.buildPythonApplication rec {
      '';
     license = licenses.isc;
     maintainers = with maintainers; [ Flakebi ];
+    broken = true;
```

Could be helpful to add a comment about the issue.

```suggestion
   # No support for ruamel.yaml > 0.17.21 
   # https://github.com/wwkimball/yamlpath/issues/217
    broken = true;
```